changelog/3175.feature

```diff
@@ -0,0 +1 @@
+Enable live logs is log_cli_level is passed/set explicitly
```

Comment:
I'm not sure about this, I can imagine someone setting `log_cli_level` in their `pytest.ini` file so that they can enable/disable live logging in the command line without having to also pass the log level each time...

@thisch what do you think?

Reply:
In that case I think we need a CLI flag to enable live logs.

Reply:
We have the `log_cli` configuration option, which can be changed directly in the command-line: `-o log_cli=True`. 😉

Reply:
I'm not sure if setting ``log_cli_level`` on the command line or in an ini file should automatically set ``log_cli`` to True. I would prefer if we have a similar shortcut for live logging, like we have for disabling stream capturing (``-s``).

Reply:
@thisch `--log-cli`? `--logs-stream`?

Reply:
I'm OK about having a `--log-cli` flag, although it is so close to `-o log_cli=true` that I'm not sure it adds much. What about having a short option for it instead? And what that would be?

Reply:
Maybe ``-S`` in analogy to ``-s``?

Reply:
Too close IMHO... @RonnyPfannschmidt are you 👎 on having a command-line for this, as you mentioned in https://github.com/pytest-dev/pytest/pull/3190#issuecomment-363199482?